src/main/java/uk/gov/hmcts/reform/refunds/services/RefundReviewServiceImpl.java

```diff
@@ -91,11 +113,19 @@
                 );
                 if (reconciliationProviderResponseResponse.getStatusCode().is2xxSuccessful()) {
                     updateRefundStatus(refundForGivenReference, refundEvent);
+                    refundForGivenReference.setRefundApproveFlag("SENT");
```

Comment:
Please create enum for RefundApproveFlag and NotificationSentFlag values instead of hardcoding 
Update it in all files